Source/com/drew/imaging/FileType.java

```diff
@@ -22,32 +22,54 @@
 
 /**
  * Enumeration of supported image file formats.
+ *
+ * MIME Type Source: https://www.freeformatter.com/mime-types-list.html
```

Comment:
The IANA maintains the "official" set of these names, so probably also worth including a link to https://www.iana.org/assignments/media-types/media-types.xhtml